themes/default/content/registry/packages/kubernetes/how-to-guides/managing-resources-with-server-side-apply.md

```diff
@@ -960,3 +960,264 @@ resources:
 {{% /choosable %}}
 
 {{< /chooser >}}
+
+## Handle Field Conflicts on Existing Resources
+
+After a resource is deployed to the cluster, it is possible for other controllers to start managing it. Thanks
+to the power of SSA, it is possible to see a detailed record of which controller last changed every field for
+the resource. An additional benefit of this fine-grained tracking is the ability to deliberately resolve conflicts
+rather than inadvertently overwriting changes made by another controller. Let's look at some common examples of field
+conflicts, and how to resolve them.
+
+### Cert Manager
+
+Cert-manager is a common example of an operator that mutates other cluster resources. Consider the following sequence
+of events:
+
+1. Deploy the cert-manager operator.
+2. Deploy a `ValidatingWebhookConfiguration` resource "X" using Pulumi with the `cert-manager.io/inject-ca-from`
+annotation set.
+3. The cert-manager operator updates "X" by setting the `webhooks[*].clientConfig.caBundle` field(s). This field is now
+managed by cert-manager rather than by Pulumi, and doesn't match Pulumi's configuration.
+4. Run another Pulumi update, and the field conflict is detected.
+
+Pulumi provides a way to ignore changes for particular fields using the [ignoreChanges resource option](/docs/intro/concepts/resources/options/ignoreChanges/).
+Set this resource option on resource "X" to tell Pulumi to ignore changes to this field since another controller is
+managing it.
+
+{{< chooser language "typescript,python,go,csharp,java,yaml" >}}
+
+{{% choosable language javascript %}}
+
+{{% /choosable %}}
+{{% choosable language typescript %}}
+
+```typescript
+const res = new MyResource("res",
```

Comment:
Should `MyResource` be `ValidatingWebhookConfiguration` to match the prose description above, or have I misunderstood that description?

Reply:
I was trying to leave the example a bit more generic since it applies to resources other than the `ValidatingWebhookConfiguration`, but perhaps that is confusing paired with the concrete example in the prose?